src/sample.py

```diff
@@ -2,21 +2,29 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function
+
+from os import environ
+environ['LD_LIBRARY_PATH'] = '/usr/local/lib/girepository-1.0'
+environ['GI_TYPELIB_PATH'] = '/usr/local/lib/girepository-1.0'
```

Comment:
This is only needed if HarfBuzz is installed there, which is not always the case.